### ISSUE TITLE
parser is adding an extra player 

### ISSUE THREAD

**Issue by ktian9** [pnxenopoulos/csgo#95]
**Describe the bug**
Following the code from the example notebook for analyzing a csgo demo, 

```
    demo_parser = DemoParser(demofile=demName,
    demo_id=demName.split(".")[0], parse_rate=128)

    data = demo_parser.parse()
    data_df = demo_parser.parse(return_type="df")
    print(csgo.analytics.stats.player_box_score(data_df["damages"], data_df["flashes"], data_df["grenades"], data_df["kills"], data_df["rounds"], data_df["weaponFires"]))
```

I supplied the demo located at here:
https://demos-us-central1.faceit-cdn.net/csgo/1-8a57ec16-beb2-4ab5-bd2e-37419211b4d2-1-1.dem.gz

and I ran into a TypeError exception
![image](https://user-images.githubusercontent.com/55895748/147797824-0bef25e5-ec74-482d-8a3c-fe220ff78515.png)

and when going to the kast function and inserting a print statement here:

![image](https://user-images.githubusercontent.com/55895748/147797847-ace27f69-4712-4f76-9710-7f701db0016c.png)

I am able to see what seems like an extra player is being added somewhere "None", which should not be happening

![image](https://user-images.githubusercontent.com/55895748/147797851-8613b3b7-dc0b-4820-b86f-47c93f7b3fe1.png)

**Comment by pnxenopoulos** [comment]
One thing we could do is to replace None with "None" or "World". Something to indicate that this has no attacker.

**Comment by pnxenopoulos** [comment]
Should be fixed in #102
